cmd/earthly/org_cmds.go

```diff
@@ -56,6 +56,37 @@ func (app *earthlyApp) orgCmds() []*cli.Command {
 
 func (app *earthlyApp) orgCmdsPreview() []*cli.Command {
 	return []*cli.Command{
+		{
+			Name:        "members",
+			Aliases:     []string{"member"},
+			Usage:       "Add and modify organization members",
+			Description: "Add and modify organization members",
+			UsageText:   "earthly org [--org <organization-name>] members (ls|update|rm)",
+			Subcommands: []*cli.Command{
+				{
+					Name:        "ls",
+					Aliases:     []string{"list"},
+					Usage:       "List organization members and their permission level",
+					Description: "List organization members and their permission level",
+					UsageText:   "earthly org [--org organization] members ls",
+					Action:      app.actionOrgMemberList,
+				},
+				{
+					Name:        "update",
```

Comment:
Should this perhaps be called `update-permission`?

I'm thinking that there will be additional things to update about a member in the future and it might be a bit difficult if the user had to specify everything about the member on the commandline, when they probably want to update only a thing or two.

Another (possibly better) alternative might be to use a flag: `earthly update --permission <permission> <email>` - this way only the properties mentioned via flags are updated.

Reply:
Could do. I was thinking that there may be more fields we may need to support--as you mentioned. I think I prefer the flag approach. Good idea.